au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TrackItem.qml

```diff
@@ -157,11 +163,9 @@ ListItemBlank {
 
             FlatButton {
                 Layout.fillWidth: true
-
-                // iconCode
-
+                Layout.preferredHeight: 24
```

Comment:
The code becomes more readable if separate properties value settings ​​into groups using empty lines